lib/board.rb

```diff
@@ -0,0 +1,55 @@
+class Board
+
+	attr_accessor :cells
+
+	def initialize
+		reset!
```

Comment:
For some reason, the indentations/spacing change when I push to Github. My indentations should be two spaces. Wondering if this is a common thing... or just me?